storage/gc_queue_test.go

```diff
@@ -304,6 +305,13 @@ func TestGCQueueIntentResolution(t *testing.T) {
 	const now int64 = 48 * 60 * 60 * 1E9 // 2d past the epoch
 	tc.manualClock.Set(now)
 
+	// TODO: why is this needed to trigger intent resolution?
```

Comment:
@bdarnell @tschottdorf this test used to work without these two puts because store.go used to write the root zone config, and so did replica_test. Without both puts, the test fails with `non-nil Txn after GC for key "0-00000"`